### PR TITLE
Adds a OOC object information button

### PR DESCRIPTION
**Pull request by Hughgent**
:cl: Hughgent
add: OOC information about objects can now be set and viewed with the new verb.
/:cl:

[why]: Another little bit of automation for reducing admin helps and mentor helps. good examples might include.
- A list of attachments that fit on this weapon.
- How a person should interact with an object in clicks, click and drag, intents.
- Humorous interjections.
- A recommended playstyle.
- various controls.

image
image
image

### CODE REVIEW

File: code/game/objects/items/tools/surgery_tools.dm [hunk 1]
```diff
@@ -140,13 +147,15 @@
 /obj/item/tool/surgery/bonegel
 	name = "bone gel"
 	icon_state = "bone-gel"
+	OOC_info = "Use to repair bones. close a ribcage or skull by 1. retractor then 2. bone gel, fix a bone by 1. bone gel 2. bone setter. A broken bone cannot be fixed if that body part has more than ~30 brute damage. Give your patient Bicaridine."
```
Comment (Rohesie) [inline code/game/objects/items/tools/surgery_tools.dm]:
```suggestion
	OOC_info = "Use to repair bones on an open incision through 1. Bone gel 2. Bone Setter. A broken bone cannot be fixed if that body part has more than ~30 brute damage. Give your patient Bicaridine."
```

File: code/game/objects/items/tools/surgery_tools.dm [hunk 2]
```diff
@@ -156,6 +165,7 @@
 /obj/item/tool/surgery/bonesetter
 	name = "bone setter"
 	icon_state = "bone setter"
+	OOC_info = "Used to repair bones. fix a bone by 1. applying bone gel. 2. using the bone setter. A broken bone cannot be fixed if that body part has more than ~30 brute damage. Give your patient Bicaridine."
```
Comment (Rohesie) [inline code/game/objects/items/tools/surgery_tools.dm]:
```suggestion
	OOC_info = "Used to repair bones. fix a bone by 1. applying Bone Gel. 2. using the Bone Setter. A broken bone cannot be fixed if that body part has more than ~30 brute damage. Give your patient Bicaridine."
```

File: code/game/objects/items/toys/toys.dm [hunk 3]
```diff
@@ -460,6 +460,7 @@
 	w_class = 1
 	icon_state = "farwaplush"
 	var/last_hug_time
+	OOC_info = "This is literally the only doll on the ship. guard it with your life."
```
Comment (Rohesie) [inline code/game/objects/items/toys/toys.dm]:
```suggestion
	OOC_info = "This is literally the only doll on the ship. Guard it with your life."
```

File: code/game/objects/objs.dm [hunk 4]
```diff
@@ -19,6 +19,7 @@
 
 	var/igniting = FALSE	//Whether it ignites on impact
 	var/item_fire_stacks = 0	//How many fire stacks it applies
+	var/OOC_info = "There is no OOC Information about this object."		//OOC information about the object, accessible via the objects tab. If you've got something to add, please make a PR and add 'OOC_info = ' to the object with your information.
```
Comment (Rohesie) [inline code/game/objects/objs.dm]:
```suggestion
	var/OOC_info = "There is no OOC Information about this object. Feel free to contribute with your own descriptions through the wiki."		//OOC information about the object, accessible via the objects tab. If you've got something to add, please make a PR and add 'OOC_info = ' to the object with your information.
```

File: code/game/objects/structures/watercloset.dm [hunk 5]
```diff
@@ -352,6 +352,7 @@
 	name = "sink"
 	icon = 'icons/obj/watercloset.dmi'
 	icon_state = "sink"
+	OOC_info = "Used to clean items of filth such as blood, dirt and, oil. clicking this with an empty hand will remove all the germs from you and will help prevent infections during surgery."
```
Comment (Rohesie) [inline code/game/objects/structures/watercloset.dm]:
```suggestion
	OOC_info = "Used to clean items of filth such as blood, dirt and, oil. Clicking this with an empty hand will remove all the germs from you and will help prevent infections during surgery."
```

File: code/game/objects/objs.dm [hunk 6]
```diff
@@ -28,6 +29,14 @@
 	. = ..()
 	GLOB.object_list -= src
 
+/obj/verb/OOC_info()
+	set name = "OOC object information"
+	set desc = "Reveals some Out of Character knowledge about an item. a mini-mentor really."
+	set category = "Object"
+	set src in range(1)
```
Comment (Rohesie) [inline code/game/objects/objs.dm]:
```suggestion
	set src in view(1)
```

File: code/game/objects/objs.dm [hunk 7]
```diff
@@ -28,6 +29,14 @@
...
+	to_chat(usr, "[src.OOC_info]")
```
Comment (Rohesie) [inline code/game/objects/objs.dm]:
```suggestion
	to_chat(usr, "[OOC_info]")
```

File: code/game/objects/items/tools/surgery_tools.dm (hunk 1 above)
Comment (Hughgent) [reply code/game/objects/items/tools/surgery_tools.dm]: Have to include the skull and ribcage closing bits, else we'll have some unhappy campers.

**Comment by Hughgent** [comment]
Probably going to try the Baystation codex idea, it's a bit self correcting in it's descriptions.

**Comment by Hughgent** [comment]
#694 is the way forward now.

**Pull request closed without merging**